README.md

```diff
@@ -1 +1,137 @@
-# sc-pipeline-management
\ No newline at end of file
+# sc-pipeline-management
+
+## Current status
+
+* Runs in a Jupyter Notebook on local machines
+* Runs in command line via Python script
+
+## Purpose
+
+* To automate the process of generating single cell pipeline input parameters from and POSTing pipeline output data to IGVF portal
+
+## Requirement
+
+### 1. Python packages
+
+* Python 3.11
+* igvf-client==63.0.0
```

Comment:
Add all the independencies into requirements.txt. Can learn from example here: https://github.com/IGVF-DACC/checkfiles/blob/dev/src/checkfiles/requirements.txt